appeals/web/src/server/appeals/appeal-details/interested-party-comments/add-ip-comment/add-ip-comment.controller.js

```diff
@@ -78,7 +78,7 @@ export async function renderUpload(request, response) {
 export async function postUpload(request, response) {
 	const { currentAppeal } = request;
 
-	request.currentFolder = await getAttachmentsFolder(request.apiClient, currentAppeal.id);
+	request.currentFolder = await getAttachmentsFolder(request.apiClient, currentAppeal.appealId);
```

Comment:
that's a footgun of an id

Reply:
Yeah 😄 I think a better convention would be `.id` is always the ID of the thing itself, and `.somethingId` is for foreign key relations.